File: sum.java
import java.util.Scanner;
public class sum {

    public static int sumOfArray(int[] arr, int n){
        int sum = 0;
        for(int i = 0; i < n; i++){
            sum += arr[i];
        }
        return sum;
    }
    public static void main(String[] args) {
        
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter the range of the Array:");
        int n = scan.nextInt();
        int arr[] = new int[n];
        System.out.println("Enter the elements of the Array:");
        for(int i = 0; i < n; i++){
            arr[i] = scan.nextInt();
        }
        scan.close();

        System.out.println("The sum of the array is :" + sumOfArray(arr, n));



        
    }
}

File: sum.cpp
#include <iostream>
using namespace std;

int sumOfArray(int arr[], int n)
{
    int sum = 0;

    for(int i = 0; i < n; i++)
    {
        sum += arr[i];
    }

    return sum;
}


int main()
{
    int arr[200], i , n , sum;
    cout << "Enter the size of the array:";
    cin >> n ;
    cout << "Enter the elements of the array:";
    for(i = 0; i < n ; i++){
        cin >> arr[i];
    }

    sum = sumOfArray(arr, n);
    cout << "The sum of the Array is :" << sum ;
    return 0;
}
    
    